### ISSUE TITLE
Exported Model - Different from test mode

### ISSUE THREAD

**Issue by groot-1313** [affinelayer/pix2pix-tensorflow#112]
I have exported the model using the `export` mode, and I am using the `process-local.py` script to test the exported model. I noticed that the images predicted by `pix2pix.py` in `test mode` are different from the images predicted by the `exported` model using `process-local.py`. Can someone please shed light upon this?

**Comment by mkarki2** [comment]
Try predicting the same images in test mode twice. They will probably be different slightly. I think that's the expected behavior. (I could be wrong.)

**Comment by groot-1313** [comment]
That is because of dropout, and the difference is very slight. The difference I am noticing is huge.

**Comment by ablenesi** [comment]
Are you using an older version? There is no `precess-local.py` in the repository. Are you referring to `server/tools/export-checkpoint.py`

**Comment by lindseycarr1977** [comment]
I'm noticing the same thing except running test mode in pix2pix.py. The output is so far from the training images (I'm doing colorization - so I wasn't sure if I had the test images in the wrong format - but i've tried every which way of formatting them)

**Comment by groot-1313** [comment]
Hi, try RGB2BGR. That worked for me.

**Comment by FrankNiemeyer** [comment]
@ablenesi The script seems to have been deleted for some reason. You can find it here:

https://github.com/affinelayer/pix2pix-tensorflow/blob/a92fdb3bc3c5e2ea3cb7c53e2cb955851fbcc6d2/server/tools/process-local.py

**Comment by ablenesi** [comment]
@FrankNiemeyer Thanks! After a little more digging I found out that all of the mentioned code was removed in favor of serving implementation with deeplearnjs.

**Comment by franciscorba** [comment]
@FrankNiemeyer I download it but got an error when tried to run it. Maybe the model is not saved in the same way as before.
